docs/internal-developers/testing/releases/911.md

```diff
@@ -0,0 +1,262 @@
+
+4. Choose other option
+
+- **Expected**: previous filter is replaced by the new one, page refreshes, filter is applied: visible in Active Filters and Products are filtered out
+
+#### Dropdown component - multiple
+
+1. Change settings of Filter by Rating to: multiple / dropdown
+2. Open dropdown
+
+- **Expected**: dropdown has rating options to choose from
+
+3. Choose one option
+
+- **Expected**: page refreshes, filter is applied: visible in Active Filters and Products are filtered out
+
+4. Choose other option
+
+- **Expected**: new filter is added to the new one, page refreshes, both filters ("or") is applied: visible in Active Filters and Products are filtered out
+
+#### List component
+
+1. Change settings of Filter by Rating to: list
+2. Repeat the above for both: single and multiple
+
+#### Apply button
+
+1. In the Editor choose multiple Dropdown again and enable "Show 'Apply filters' button" option
+2. In the frontend choose rating filters
+
+- **Expected**: filters are not applied until "Apply" button is clicked. Once clicked page refreshes and filters are applied
+
+#### Other scenarios
+
+Try to break it! 😉
+
+### Product Query - Enable "Inherit Query from template" option ([7641](https://github.com/woocommerce/woocommerce-blocks/pull/7641))
+
+#### Site Editor
+
+1. Add the Product Query in a template (eg: Product Catalog)
+2. Check that the `Inherit Query from template` option is visible.
+3. Enable it.
+4. Be sure that all the settings that customize the query are hidden.
+5. Save.
+6. Be sure that the result generated by the Product Query is the same as the classic FSE template (there are E2E tests that check this ).
+7. Edit the Product Catalog and disable the `Inherit Query from template` option.
+8. Ensure all the settings that customize the query are visible.
+
+#### Post/Page Editor
+
+1. Add the Product Query in a template (eg: Product Catalog)
+2. Check that the `Inherit Query from template` option isn't visible.
+
+### Fix the incorrect layout of Rating and Price in Classic Template and Products block ([7932](https://github.com/woocommerce/woocommerce-blocks/pull/7932))
+
+Prerequisites:
+
+- make sure you have at least one product _with_ rating
+
+Steps:
+
+1. Go to Editor and add blocks:
+
+- Filter by Rating
+- All Products,
+- Products
+- Classic Template
+
+2. In the frontend check the above blocks
+
+**Expected:**
+
+- The layout of product in Products and Classic Template is as on the screenshots above
+- The layout of the Filter by Rating and All Products doesn't change
+
+### Product Elements: Fix block settings ([7914](https://github.com/woocommerce/woocommerce-blocks/pull/7914))
+
+1. Create a page and add the **Products (Beta)** block.
+2. Add **Button**, **Price** and **Rating** product elements.
+3. Customize via block settings:
+
+- **Product Price**: `background color` and `appearance`.
+- **Product Button**: `font-weight` and `padding`.
```

Comment:
I can't see `font-weight` here: :confused: 
```suggestion
- **Product Button**: `padding`.
```

Reply:
I think that it refers to the `Typography` section.

![image](https://user-images.githubusercontent.com/4463174/207659379-6c4b1aa5-936e-4fbd-9949-2da7e2d2704d.png)

Reply:
I update these lines in this way:

```
- **Product Price**:  `Color > background`, `Color > text ` and `Typography > Appearance`.
- **Product Button**: `Typography > font-weight` and `Dimension > padding`.
- **Product Rating**: `Typography > font-size`, `Dimension > margin`, `Color > text`.

Reply:
LGTM :+1: